src/script/hydra-create-user

```diff
@@ -27,6 +28,14 @@ renamed.
 
 * Specifying Passwords
 
+** Interactively
+
+Pass `--password-prompt` to collect the password on stdin.
```

Comment:
IMO, this should also note that it will be hashed with argon2 (and also that it's not stored in plaintext, maybe?).